doc/src/modules/stats.rst

```diff
@@ -116,12 +132,25 @@ Interface
 .. autoclass:: Variance
 .. autofunction:: covariance
 .. autoclass:: Covariance
+.. autofunction:: coskewness
+.. autofunction:: median
 .. autofunction:: std
 .. autofunction:: sample
 .. autofunction:: sample_iter
+.. autofunction:: factorial_moment
+.. autofunction:: kurtosis
+.. autofunction:: skewness
+.. autofunction:: correlation
 .. autofunction:: sympy.stats.rv.sampling_density
 .. autofunction:: sympy.stats.rv.sampling_P
 .. autofunction:: sympy.stats.rv.sampling_E
+.. autoclass:: Moment
```

Comment:
New lines missing after, `:` in the following code,
https://github.com/sympy/sympy/blob/4b23967d96ec120dd479fce8e04851a6323f86c3/sympy/stats/symbolic_probability.py#L580-L594

How the docs look like right now?
```python
To evaluate the result of Moment use doit: >>> M.doit() mu**3 - 3*mu**2 + 3*mu*sigma**2 + 3*mu - 3*sigma**2 - 1

Rewrite the Moment expression in terms of Expectation: >>> M.rewrite(Expectation) Expectation((X - 1)**3)

Rewrite the Moment expression in terms of Probability: >>> M.rewrite(Probability) Integral((x - 1)**3*Probability(Eq(X, x)), (x, -oo, oo))

Rewrite the Moment expression in terms of Integral: >>> M.rewrite(Integral) Integral(sqrt(2)*(X - 1)**3*exp(-(X - mu)**2/(2*sigma**2))/(2*sqrt(pi)*sigma), (X, -oo, oo))
```

Reply:
Yes, it will require a new line. Fixed it!